tensorflow_datasets/core/lazy_imports_lib.py

```diff
@@ -154,3 +225,5 @@ def test_foo(cls):
 
 
 lazy_imports = LazyImporter  # pylint: disable=invalid-name
+sys.path_hooks.append(LazyImporter)
```

Comment:
This adds the imported globally, while we only want to add it within the lazy context manager.